templates/_call-stack-item.php

```diff
@@ -2,40 +2,28 @@
 /* @var $file string|null */
 /* @var $line int|null */
 /* @var $class string|null */
-/* @var $method string|null */
+/* @var $function string|null */
 /* @var $index int */
 /* @var $lines string[] */
 /* @var $begin int */
 /* @var $end int */
 /* @var $args array */
-/* @var $this \Yiisoft\ErrorHandler\HtmlRenderer */
-
-$html = <<<HTML
-IDE
-<svg class="icon icon--new-window" focusable="false" aria-hidden="true" width="16" height="16">
-    <use href="#new-window"></use>
-</svg>
-HTML;
+/* @var $this \Yiisoft\ErrorHandler\Renderer\HtmlRenderer */
 ?>
-<li class="<?= ($index === 1 || !$this->isCoreFile($file)) ? 'application' : '' ?> call-stack-item"
-    data-line="<?= (int) ($line - $begin) ?>">
+<li class="<?=!empty($lines) ? 'application' : '' ?> call-stack-item" data-line="<?= (int) ($line - $begin) ?>">
     <div class="element-wrap">
         <div class="flex-1">
             <?php if ($file !== null): ?>
                 <span class="file-name">
-                    <?= sprintf('%d. %s', (int) $index, 'in ' . $this->htmlEncode($file))  ?>
+                    <?= "{$index}. in {$this->htmlEncode($file)}" ?>
                 </span>
             <?php endif ?>
 
-            <?php if ($this->traceLine !== '{html}') : ?>
```

Comment:
It was previously used to have an ability to use custom template. A typical use-case was creating IDE-specific links for when trace file is clicked, it is opened right in the IDE.

Reply:
Returned, renamed the property to the `$traceHeaderLine`.